package.json

```diff
@@ -13,9 +13,9 @@
                 "carrier": "0.1.10",
                 "ipaddr.js": "0.1.1",
                 "jsprim": "0.5.0",
-                "libmanta": "1.4.0",
-                "marlin": "git+https://github.com/joyent/manta-marlin.git#57150b778bc5cf41a7d366de0ea7230648866b05",
-                "manta": "5.2.1",
+                "libmanta": "git+https://github.com/joyent/node-libmanta.git#v1.4.4",
```

Comment:
Is 1.4.4 not in NPM?  Or am I missing some other subtlety?

Reply:
Correct. In joyent/node-libmanta#13 Trent decided to stop publishing. Not entirely sure why, but not something I wanted to revisit right now.